### PR TITLE
Add fabric version 1.4.4 to start fabric script

### PR DESCRIPTION
**Pull request by Ram-srini**
Add fabric version 1.4.4 to start fabric script

Signed-off-by: Ramakrishna Srinivasamurthy <ramakrishna.srinivasamurthy@intel.com>

### CODE REVIEW

File: scripts/start_fabric.sh [hunk 1]
```diff
@@ -44,12 +44,12 @@ if [[ ! $(docker ps --format '{{.Names}}' |grep "peer*") ]]; then
...
     docker rm -f cli
     docker run -dit --name cli --network minifab -v /var/run/docker.sock:/var/run/docker.sock -v $(pwd)/vars:/vars \
--v $(pwd)/vars/chaincode:/go/src/github.com/chaincode \
+-v $(pwd)/vars/chaincode:/opt/gopath/src/github.com/chaincode \
 -e "http_proxy=$http_proxy" -e "https_proxy=$https_proxy" \
 -e "no_proxy=$no_proxy,localhost,orderer3.example.com,orderer2.example.com,orderer1.example.com,peer2.org1.example.com,peer1.org1.example.com,peer2.org0.example.com,peer1.org0.example.com" \
 -e "HTTP_PROXY=$http_proxy" -e "HTTPS_PROXY=$https_proxy" \
 -e "NO_PROXY=$no_proxy,localhost,orderer3.example.com,orderer2.example.com,orderer1.example.com,peer2.org1.example.com,peer1.org1.example.com,peer2.org0.example.com,peer1.org0.example.com" \
-hyperledger/fabric-tools:2.0
+hyperledger/fabric-tools:1.4.4
```
Comment (manojgop) [inline scripts/start_fabric.sh]: You could use a FABRIC_VERSION variable and set it once and use it at multiple places in this script.

Comment (Ram-srini) [reply scripts/start_fabric.sh]: Taken care of this.

File: scripts/start_fabric.sh [hunk 2]
```diff
@@ -14,8 +14,33 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+FABRIC_VERSION=1.4.4
+CHAIN_CODE_VERSION=1.0
```
Comment (manojgop) [inline scripts/start_fabric.sh]: Please add a comment why we are using Fabric 1.4.4 version in Avalon as default instead of 2.0

**Comment by Ram-srini** [comment]
@litong01 Please review these changes.

**Pull request merged**